CONTRIBUTING.rst

```diff
@@ -83,12 +83,16 @@ Releases are created using `manual GitHub workflows
 <https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow>`_.
 As a maintainer, follow these steps:
 
-1. Create a new branch
-2. Bump the version defined in ``skops/__init__.py``
-3. Git grep for any TODO's that need fixing before the release (e.g.
+1. Check and update the ``CHANGES.md``
+2. For a major release, create a new branch with the name "0.version.X", e.g.
+   "0.2.X". This branch will have all tags for all releases under 0.2.
+3. Bump the version defined in ``skops/__init__.py``
+4. Git grep for any TODO's that need fixing before the release (e.g.
    deprecations)
-4. Update the ``CHANGES.md``
 5. Create a PR with all the changes and have it reviewed and merged
+6. Create a tag with the format "v0.version", e.g. "v0.2", and push it to the
+   remote repository. Use this tag for releasing the package. If there is a
+   minor release under the same branch, it would be "v0.2.1" for example.
```

Comment:
Numbers below this need to be incremented (6->7 etc.). I think that replacing the numbers with `#.` numbers them automatically.